app/controllers/media-controller.js

```diff
@@ -9,6 +9,10 @@ const request = require( 'request' );
 const express = require( 'express' );
 const router = express.Router();
 const debug = require( 'debug' )( 'media-controller' );
+const { RequestFilteringHttpAgent } = require( 'request-filtering-agent' );
+
+const config = require('../../config/config');
+const requestFilteringOptions = config.IPfiltering;
```

Comment:
Is there a file missing perhaps? I don't see where `config.IPFiltering` is populated.

Reply:
It's being populated in the `config.json`, it's where the options for `request-filtering-agent` are -- I've added it to the `default-config.json`.